src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +57,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    
+    @Override
+    public String getPrintableString(){
```

Comment:
Correct implementation of the interface. Good use of the constant for the prefix.